content/basics/operations/troubleshooting-provisioning-errors/index.md

```diff
@@ -33,15 +33,35 @@ To get started, follow steps shown next:
+:::
+1. Use the AWS CLI to create the stack. The required parameter `--template-body` have been pre-filled for you.
+:::code{language=shell showLineNumbers=false showCopyAction=false}
+aws cloudformation create-stack --stack-name cfn-workshop-troubleshooting-provisioning-errors --template-body file://sqs-queues.yaml --disable-rollback
+:::
+1. If the `create-stack` command was successfully sent, CloudFormation will return `StackId`.
+:::code{language=shell showLineNumbers=false showCopyAction=false}
+"StackId": "arn:aws:cloudformation:us-east-1:123456789012:stack/cfn-workshop-troubleshooting-provisioning-errors/739fafa0-e4d7-11ed-a000-12d9009553ff"
+:::
+ 1. Open the **[AWS CloudFormation](https://console.aws.amazon.com/cloudformation)** console in a new tab and check if the stack status is **CREATE_FAILED**.
+::::
 
+::::tab{id="local" label="Local development"}
 1. Navigate to the [AWS CloudFormation Console](https://console.aws.amazon.com/cloudformation/), and [choose a Region](https://docs.aws.amazon.com/awsconsolehelpdocs/latest/gsg/select-region.html) you wish to use.
-2. From **Create stack**, choose **With new resources (standard)**.
-3. From **Prepare template**, choose **Template is ready**.
-4. From **Template source**, choose **Upload a template file**. Choose the `sqs-queues.yaml` template mentioned earlier, and then choose **Next**.
-5. Specify a stack name: for example, `troubleshoot-provisioning-errors-workshop`. On the same page, accept the default value for the `QueueNamePrefix` parameter, and choose **Next**.
-6. In **Stack failure options**, select **Preserve successfully provisioned resources**. Choose **Next**.
-7. Choose **Create stack** in the next page.
-8. Refresh the stack creation page until you see your stack in the `CREATE_FAILED` status.
+1. From **Create stack**, choose **With new resources (standard)**.
+1. From **Prepare template**, choose **Template is ready**.
+1. From Prepare template, choose Template is ready.
+1. From Template source, choose Upload a template file. Choose the `sqs-queues.yaml` template mentioned earlier, and then choose Next.
+1. Specify a stack name: for example, `cfn-workshop-troubleshooting-provisioning-errors`. On the same page, accept the default value for the `QueueNamePrefix` parameter, and choose **Next**.
+1. In **Stack failure options**, select **Preserve successfully provisioned resources**. Choose **Next**.
+1. Choose **Create stack** in the next page.
+1. Refresh the stack creation page until you see your stack in the `CREATE_FAILED` status.
+::::
+:::::
 
 The stack creation has failed, because of the error mentioned earlier. Select the name of your stack from the list (for example, `troubleshoot-provisioning-errors-workshop`): in the **Resources** tab, note that the `DeadLetterQueue` resource is in the `CREATE_COMPLETE` status, and the `SourceQueue` resource is in `CREATE_FAILED` status, along with a relevant error in the **Status reason** column.
```

Comment:
```suggestion
The stack creation has failed, because of the error mentioned earlier. Select the name of your stack from the list (for example, `cfn-workshop-troubleshooting-provisioning-errors`): in the **Resources** tab, note that the `DeadLetterQueue` resource is in the `CREATE_COMPLETE` status, and the `SourceQueue` resource is in `CREATE_FAILED` status, along with a relevant error in the **Status reason** column.
```